cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -366,6 +376,73 @@ public KafkaAssemblyOperator(Vertx vertx, PlatformFeaturesAvailability pfa,
         return chainPromise.future();
     }
 
+    Future<Void> reconcileWithoutZoo(ReconciliationState reconcileState)  {
```

Comment:
We're going to have to live with both modes for a while, during which there will be continued churn to the reconciliation process. So I wonder if it would be better to pass the mode flag and branch within the compose steps.